index.html

```diff
@@ -37,6 +37,119 @@ <h2 class="font connect">LET'S CONNECT </h2>
+      <ul>
+        <li class="canopy">CANOPY</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">Back End Dev</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li><button class="lang">html</button></li>
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+    <div class="card">
+      <img class="work-img" src="images/Snapshoot3.png" alt="">
+      <h2>Tonic</h2>
+      <ul>
+        <li class="canopy">CANOPY</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">Back End Dev</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li><button class="lang">html</button></li>
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+    <div class="card">
+      <img class="work-img" src="images/Snapshoot4.png" alt="">
+      <h2>Multi-Post Stories</h2>
+      <ul>
+        <li class="canopy">CANOPY</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">Back End Dev</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li><button class="lang">html</button></li>
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+  </section>
+  <section class="about font align">
+    <h2 class="intro font">About Myself</h2>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="font connect">LET'S CONNECT </p>
+    <ul class="vectors">
+      <li><a href=""><img src="images/twiter.svg" alt=""></a></li>
+      <li><a href=""><img src="images/N.svg" alt=""></a></li>
+      <li><a href=""><img src="images/M.svg" alt=""></a></li>
+      <li><a href=""><img src="images/github.svg" alt=""></a></li>
+      <li><a href=""><img src="images/two.svg" alt=""></a></li>
+    </ul>
+    <button class="btn">Get my resume</button>
+    <ul class="skills">
+      <li>
+        <h2>Languages</h2>
```

Comment:
- [ ] [Required] Please, Add Drop-down Icon after the `Languages` heading as shown in the Figma design 
![Lang_Screenshot from 2022-05-28 20-20-31](https://user-images.githubusercontent.com/24191094/170836102-0579aedb-9eb8-4eb9-95f1-1f5ea6772711.png)